File: examples/csharp/valid.cs
using System;

public class Student
{
    public string Name { get; set; } = "Ayesha";
    public int Age { get; set; } = 20;

    public bool IsAdult()
    {
        return Age >= 18;
    }

    public static void Main()
    {
        var student = new Student();
        Console.WriteLine(student.Name);
    }
}

File: examples/java/valid.java
public class Student {
    private String name = "Ayesha";
    private int age = 20;

    public boolean isAdult() {
        return age >= 18;
    }

    public static void main(String[] args) {
        Student student = new Student();
        System.out.println(student.name);
    }
}

